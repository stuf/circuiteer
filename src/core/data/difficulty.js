/**
 * @type {Game.IDifficulty}
 */
export const Difficulty = {
  EASY: 'easy',
  MEDIUM: 'medium',
  HARD: 'hard',
  VERY_HARD: 'veryHard',
};
