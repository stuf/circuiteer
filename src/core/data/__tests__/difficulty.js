import { Difficulty } from '../difficulty';

test('Difficulty', () => {
  expect(Object.keys(Difficulty)).toHaveLength(4);

  expect(Difficulty).toMatchSnapshot();
});
