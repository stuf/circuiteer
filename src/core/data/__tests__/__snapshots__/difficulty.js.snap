// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Difficulty 1`] = `
Object {
  "EASY": "easy",
  "HARD": "hard",
  "MEDIUM": "medium",
  "VERY_HARD": "veryHard",
}
`;
